docs/jam/jam/literals/integers.rst

```diff
@@ -5,16 +5,26 @@ Integers
 
 A integer is a whole number of the builtin type ``Int``.
 
+As detailed in the :ref:`builtin library <jam-builtins-numerical>`,
```

Comment:
I think this has less to do with the actual syntax of integers and more with types. There should be a link, but maybe less explanation? Like: `See the builtin library for more detail on integer types`.